vendor/github.com/stretchr/testify/vendor/github.com/stretchr/objx/codegen/types_list.txt

```diff
@@ -0,0 +1,20 @@
+Interface,interface{},"something",nil,Inter
```

Comment:
You are right, that is quite strange, because they are not updated in the vendor.json file?!? Glad you cleaned that up.